includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -197,6 +258,13 @@ public static function get_upe_enabled_payment_method_ids( $force_refresh = fals
 			}
 		}
 
+		// Save in options, so we can use this as a fallback.
+		if ( $force_refresh ) {
+			$stripe_settings = WC_Stripe_Helper::get_stripe_settings();
+			$stripe_settings['upe_checkout_experience_accepted_payments'] = $enabled_payment_method_ids;
+			WC_Stripe_Helper::update_main_stripe_settings( $stripe_settings );
+		}
+
```

Comment:
Not happy with this. This means we only get a copy when `$force_refresh` is true, ie. when user visits the Stripe settings page.

If we remove the check though, we will do it far too often.

Reply:
Forgot to include express checkout methods too -- Apple Pay, Google Pay, Link.